mix.exs

```diff
@@ -25,6 +25,7 @@ defmodule ApiUmbrella.Mixfile do
           applications: [
             runtime_tools: :permanent,
             alb_monitor: :permanent,
+            config_providers: :permanent,
```

Comment:
question: Does this actually have any servers that need to run? It seems like it's more of a library application which wouldn't need to be listed here, or possibly a `:temporary` application because it shuts down after loading the configuration?

Reply:
I don't _think_ it has any servers that run (though the release approach something like "start up the app and do some configuration and write it down, and then restart it" I'm still a little hazy on). My main concern was the release stripping out that code. I thought if the applications weren't mentioned, it might not be compiled into the final artifact, but I can try removing and see if it works when I deploy to dev-blue and dev-green.